client/components/data/query-preferences/index.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isFetchingPreferences } from 'state/preferences/selectors';
+import { fetchPreferences } from 'state/preferences/actions';
+
+class QueryPreferences extends Component {
+
+	componentWillMount() {
+		if ( ! this.props.isFetchingPreferences ) {
+			this.props.fetchPreferences();
+		}
+	}
+
+	render() {
+		return null;
+	}
+}
+
+QueryPreferences.propTypes = {
+	requestingSites: PropTypes.bool,
+	fetchPreferences: PropTypes.func
+};
+
+QueryPreferences.defaultProps = {
+	fetchPreferences: () => {}
```

Comment:
That is not true and I will never again omit the `defaultProps ` again.
Omitting `fetchPreferences ` here fails the tests in  `test/post-editor.jsx` because mock reduxStore there does not really work. (I know, I introduced this mock)
In my opinion, while in transition, that test does more harm then good and I believe it should be first deleted, and after finishing redux transition, rewritten as a shallow renderer.
But, that is not my call :)